### PR TITLE
Adding a `_became_false?` method

### PR DESCRIPTION
**Pull request by raelgc**
Similar to `_became_true?`, to avoid checks like `bit_changed? && !bit?`.

### CODE REVIEW

File: lib/bitfields/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Bitfields
-  Version = VERSION = "0.9.0"
+  Version = VERSION = "0.9.1"
```
Comment (grosser) [inline lib/bitfields/version.rb]: don't bump in PRs ... also this is not a patch, but a minor (new feature)

**Comment by grosser** [comment]
0.10.0 🎉

File: lib/bitfields/version.rb (hunk 1 above)
Comment (raelgc) [reply lib/bitfields/version.rb]: Sorry!

**Comment by grosser** [comment]
np, keep the PRs coming :D

**Pull request merged**